docs/src/pages/components/data-grid/components/CustomToolbarGrid.tsx

```diff
@@ -18,8 +18,8 @@ function CustomToolbar() {
 
 export default function CustomToolbarGrid() {
   const { data } = useDemoData({
-    dataSet: 'Commodity',
-    rowLength: 100,
+    dataSet: 'Employee',
```

Comment:
I would personally keep the same data set between the two demos under `### Toolbar` to not to create a distracting on what we want the developer to focus on (the `CustomToolbar`, not the change of data set)

```suggestion
    dataSet: 'Commodity',
```